### PR TITLE
Update Tizen CI docker image

### LINKED ISSUE #17028: Tizen armel Cross Checked Innerloop Build and Test failing pretty much every PR
https://ci.dot.net/job/dotnet_coreclr/job/master/job/armel_cross_checked_tizen_innerloop_prtest/

We need to remove the default trigger for this job unless/until it is made reliable.

### PR DESCRIPTION
**Pull request by gbalykov**
Update Tizen CI docker image to ubuntu 16.04

This seems to fix #17028. Related issue (https://github.com/dotnet/corefx/issues/28901)

cc @Dmitri-Botcharnikov @alpencolt

### CODE REVIEW

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by jkotas** [comment]
> @jkotas could we reenable Tizen in default CI now?

@RussKeldorph  ?

**Comment by RussKeldorph** [comment]
@gbalykov Does this also address QEMU issues like #6298?  If not, I suspect we won't be able to tolerate the spurious failures in the default CI jobs.  If you aren't sure, we can do a trial run, on the condition that if the spurious failures return, we disable again.

**Comment by gbalykov** [comment]
@RussKeldorph 

Docker image now has newer version of QEMU. I've ran Tizen tests on this PR and there seems to be no problem with them. I agree with your idea to disable tests if spurious failures return.

I'll update this PR with enabled Tizen CI in default jobs and open same PR for corefx/core-setup.

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@dotnet-bot test this please

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@dotnet-bot test this please

**Comment by gbalykov** [comment]
@dotnet-bot test this please

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by gbalykov** [comment]
@RussKeldorph, could you, please, take a look?

File: netci.groovy [hunk 1]
```diff
@@ -210,6 +210,11 @@ class Constants {
                 'Checked'
             ]
         ],
+        'Tizen': [
+            'armem': [
```
Comment (janvorli) [inline netci.groovy]: @gbalykov should this be "armel" instead of "armem"?

Comment (gbalykov) [reply netci.groovy]: @janvorli, this is simply revert of part of #17240. `armem` is used for ARM builds/runs using emulator.

Comment (janvorli) [reply netci.groovy]: Ah, ok, thank you for the clarification!

**Comment by janvorli** [comment]
@dotnet-bot test ci please

**Comment by gbalykov** [comment]
@dotnet-bot test ci please

**Comment by gbalykov** [comment]
@janvorli, I've fully reverted commit, which disabled Tizen CI, so now "test ci please" should work. However, it seems that I can't run it. Could you, please, rerun it?

**Comment by janvorli** [comment]
@dotnet-bot test ci please

**Comment by gbalykov** [comment]
@janvorli Thank you!

**Pull request merged**